src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -26,7 +27,7 @@ public SortCommand(String category) {
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.sortPersonList(category);
-        return new CommandResult(MESSAGE_SUCCESS);
+        return new CommandResult(MESSAGE_SUCCESS + category + ".");
```

Comment:
I like that you included the category, perhaps a possible alternative implementation to consider (from EditCommand.java)
`public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";`
`String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson)`
the same implementation could also be considered for the tests